controller_manager/src/controller_manager.cpp

```diff
@@ -1567,16 +1538,14 @@ void ControllerManager::activate_controllers(
     // if it is a chainable controller, make the reference interfaces available on activation
     if (controller->is_chainable())
     {
-      resource_manager_->make_controller_reference_interfaces_available(request);
+      resource_manager_->make_controller_reference_interfaces_available(controller_name);
     }
 
     if (controller->is_async())
     {
       async_controller_threads_.at(controller_name)->activate();
     }
   }
-  // All controllers activated, switching done
-  switch_params_.do_switch = false;
```

Comment:
this changes behaviour, nicht wahr?

I see it was moved into `manage_switch()` to the end

Reply:
No, it should not change behavior, but just put it in a more logical place.

Reply:
I think what you said makes sense. It should be at the end of the `manage_switch` method.